blog/_posts/2016-10-11-team-fedex++-signing-off.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Team Fedex++ Signing Off
+layout: post
+created_at: Tue October 11 2016
+permalink: blog/2016-10-11-Team-Fedex++-Signing-Off
+current: blog
+author: Mansi and Sahar
+twitter: TeamFedex
+categories: student-posts
+---
+
+
+*When you do something best in life, you don't really want to give that up.*  
+            - [_Roger Federer_](https://twitter.com/rogerfederer "Roger Federer")
+
+![Team-Fedex++](/img/blog/2016/TeamFedex++intro.jpg)    
+*image taken with proper permissions*  
+
+As appropriately quoted by Federer, when you start with something really good, it becomes so close to your heart that you don’t want to let it go. So is our connection with the RGSoC program and the community. It’s been 4 months since we became a part of it and it seems this journey had just begun a while ago.
+
+It all started with our selection to Rails Girls Summer of Code as a volunteer team to PyDSA project. Eventually, we have come to the stage where the learning process and bond with the community has become so strong that we wish to be connected to it forever. During this journey filled with so many experiences, some moments were so memorable and cherished that now it is really hard to go back and bid our goodbye.
+
+
+### **Experience working in open source** 
+- This was our first proper experience with the Open Source community.
+- We had primarily taken up the RGSoC project to get exposure to Open Source, to foray into this never ending space filled with excitement, to get to know how to code into an actual live project repo and the feeling when your PR gets merged. We learnt a lot about community bonding, for instance it is not about what you learn but what you return to the community, how even a small change or input from your side may impact someone’s life deeply.
+- We got to interact with many people as part of conferences (PyCon India) and some very inspirational ladies (PyLadies Pune Chapter)
+- It is always a challenge to handle your academics along with your passion when you are a student but through the summer program we realised the importance of being a part of the community and what the world of open source can teach you which you might never learn under the bounds of an institute.
+
+
+![Team PyDSA](/img/blog/2016/TeamFedex++pydsa.png)    
+*All images have been taken from Rails Girls Team App with permissions*   
```

Comment:
Why is this credited with "taken from Rails Girls Team app"? If one of you took a screenshot of your hangout, please use your own name as a credit.